README.md

```diff
@@ -4,3 +4,102 @@ Welcome to the example repository for integrating a custom copilot into OpenBB
 Terminal Pro. This repository provides reference implementations to help you
 create and add your own custom copilots to enhance the capabilities of Terminal
 Pro.
+
+## Requirements for Compatibility
+
+Custom copilots are expected to follow the defined schema for queries, SSE responses, and function calling. These requirements ensure seamless integration with OpenBB Terminal Pro.
+
+### Query Endpoint
```

Comment:
```suggestion
### How Terminal Pro submits requests to your Copilot
```

Reply:
I think start with this first -- explain that Terminal Pro will submit POST requests to the `/v1/query` endpoint, and that your custom copilot must respond with SSEs. 

Link to the MDN docs for SSEs (since devs might not be familiar with it).